fast64_internal/oot/oot_f3d_writer.py

```diff
@@ -601,6 +605,11 @@ def drawOOTMaterialDrawLayerProperty(layout, matDrawLayerProp, suffix):
 			i = 8 + colIndex * 3 + rowIndex
 			name = "Segment " + format(i, 'X') + " " + suffix
 			col.prop(matDrawLayerProp, "segment" + format(i, 'X'), text = name)
+		name = "Custom " + str(colIndex) + " " + suffix
```

Comment:
```suggestion
		name = "Jump to (" + str(colIndex + 1) + ") " + suffix
```
I think "custom" is a pretty bad name to use for everything (way too generic), maybe change `custom` to `customJump`